ui/packages/consul-ui/lib/startup/templates/body.html.js

```diff
@@ -66,6 +66,9 @@ ${
 {{if .NamespacesEnabled}}
   <script src="${rootURL}assets/consul-nspaces/routes.js"></script>
 {{end}}
+{{if true}}
```

Comment:
^ This `true` here needs to be replaced with something like `.HcpEnabled`, or anything that can be flipped between true/false via the backend depending on whether we are in HCP or not